samples/hello_afu/hw/rtl/afu_json_info.vh

```diff
@@ -0,0 +1,13 @@
+//
+// Generated by afu_json_mgr from cci_hello.json
+//
+
+`ifndef __AFU_JSON_INFO__
+`define __AFU_JSON_INFO__
+
+`define AFU_ACCEL_NAME "hello_afu"
+`define AFU_ACCEL_UUID 128'hc6aa954a_9b91_4a37_abc1_1d9f0709dcc3
```

Comment:
hello_afu UUID incorrect here, not the same one from json file. This one is hello_world's AFU ID and that is the reason why hello_afu test failed.

Reply:
Hello AFU ID should be 128'h1cddcf01_c266_44de_af63_9d09858310e5